pyEpiabm/pyEpiabm/utility/inverse_cdf.py

```diff
@@ -0,0 +1,63 @@
+#
+# Class of inverse CDF as is done in Covidsim code.
+#
+
+import random
+import pyEpiabm as pe
+import math
+import numpy as np
+
+
+class InverseCdf:
+    """Class of inverse cumulative distribution functions and their associated
+    methods, in a style similar to CovidSim.
+    """
+
+    def __init__(self, mean, icdf_array):
+        """Constructor Method.
+
+        :param mean: Mean of the icdf
+        :type mean: float
+        :param icdf_array: Array of quantiles of the icdf_array
+        :type icdf_array: numpy array
+        """
+        self.mean = mean
+        self.icdf_array = icdf_array
+        self.CDF_RES = pe.Parameters.instance().CDF_RES
+
+    def icdf_choose_noexp(self) -> float:
+        """Chooses a value from the input inverse cumulative distribution function,
+        following what is done in CovidSim (without exponentialisation), and
+        returns the value as a float.
+
+        :returns: chosen value among the quantiles
+        :rtype: float
+        """
+        time_steps_per_day = pe.Parameters.instance().time_steps_per_day
+        rand_num = random.random()
+        q = rand_num*self.CDF_RES
```

Comment:
```suggestion
        q = rand_num * self.CDF_RES
```
Spaces around operators are preferred